components/script/dom/htmlimageelement.rs

```diff
@@ -414,23 +414,16 @@ impl HTMLImageElement {
     pub fn areas(&self) -> Option<Vec<Root<HTMLAreaElement>>> {
         let elem = self.upcast::<Element>();
         let usemap_attr;
-        if elem.has_attribute(&LocalName::from("usemap")) {
-            usemap_attr = elem.get_string_attribute(&local_name!("usemap"));
-        } else {
-            return None;
+        if !elem.has_attribute(&LocalName::from("usemap")) {
+            return None
         }
+        usemap_attr = elem.get_string_attribute(&local_name!("usemap"));
```

Comment:
I think this would be better as
```rust
let usemap_attr = match elem.get_attribute(&ns!(), &local_name!("usemap")) {
    Some(attr) => attr,
    None => return None,
};
```